compiler/testData/codegen/box/callableReference/function/inlineConstructor.kt

```diff
@@ -0,0 +1,7 @@
+class A<T> (val result: T)
```

Comment:
...and test is redundant. there is more correct innerGenericConstuctor.kt with inners